### PR TITLE
GMP-CF-0002

### PR DESCRIPTION
**Pull request by davidlohle**
Documentation on how to apply the temporary feature to remove Consul from v1.3 CF Kit.

### CODE REVIEW

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+toplinks: yes
+sticknav: yes
+--- |-
+
+# Overview
+
+With BOSH DNS a mature product and an integral component in many BOSH
+deployments, we have begun phasing out Consul DNS with BOSH DNS. The
+Cloud Foundry Genesis Kit v1.3 is the first step in this two-step
+phase. v1.3 colocated BOSH DNS and disable Consul DNS, but does not
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: Use present tense: "colocates" and "disables"

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+url:      /docs/migrations/gmp-cf-0002/index.html
+format:   markdown
+toplinks: yes
+sticknav: yes
+--- |-
+
+# Overview
+
+With BOSH DNS a mature product and an integral component in many BOSH
+deployments, we have begun phasing out Consul DNS with BOSH DNS. The
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: reword to "replacing Consul service discovery with BOSH DNS"

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+Cloud Foundry Genesis Kit v1.3 is the first step in this two-step
+phase. v1.3 colocated BOSH DNS and disable Consul DNS, but does not
+remove Consul entirely. The result is a seamless upgrade from v1.2.X
+while also leveraging BOSH DNS for all service discovery.
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: drop "all"

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+It is our plan to fully remove Consul from the Cloud Foundry Genesis
+Kit in v1.4, which will include the deletion of all `consul_etcd`
+virtual machines. This is the second step of the two-step phase, which
+enables a zero-downtime upgrade path from Consul DNS to BOSH DNS.
+
+However, if an operator wishes to remove all Consul related services
+from their Cloud Foundry deployment without waiting for v1.4, they may
+do so with a temporary feature flag implemented just for v1.3 called
+`migrate-1.3-without-consul`. This following GMP will explain the
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: drop "following"

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+steps necesary to use this feature.
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: typo "necessary"

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+# Impact
+
+We believe the impact of this feature to be minimal on the stability
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: Drop the "believe" verbiage, and be assertive!

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+and runtime of your Cloud Foundry deployment. The decision to either
+apply or not apply this feature to your deployments will not impact
+future upgradability of this environment. No extra steps are necessary
+to maintain your Cloud Foundry environment after this GMP is
+completed.
+
+There is no downtime necessary to apply this feature.
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: replace "necessary to apply" with "incurred by applying"; we incur downtime.

File: data/pages/docs/migrations/gmp-cf-0002.yml [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+The `based on kit` line must read `cf/1.3`. If it does not, please
+upgrade your Cloud Foundry installation with the v1.3 Cloud Foundry
+Genesis Kit before starting.
+
+If your `based on kit` line reads `cf/1.3`, and you wish to remove
+Consul entirely, please continue reading this guide.
+
+## Removing Consul
+
+After ensuring the environment you wish to remove Consul on is on
```
Comment (jhunt) [inline data/pages/docs/migrations/gmp-cf-0002.yml]: "remove Consul from"

**Review by jhunt — CHANGES_REQUESTED** [review]
Overall looks good, just some typos and rewording requests.

**Comment by davidlohle** [comment]
Adjusted for feedback! 🎉

**Pull request merged**